doc/clone-from-volumesnapshot-source.md

```diff
@@ -0,0 +1,82 @@
+# Data Volume cloning from a VolumeSnapshot source
+
+## Introduction
+Current CDI PVC cloning methods offer [smart cloning](./smart-clone.md) which uses a 1:1 cloning method that creates a temporary snapshot of the source PVC for each intended clone.  
+Some storage systems are designed to scale better with a 1:N method where a single snapshot of the source PVC is used to create multiple clones.  
+One examples is ceph, where this would be about utilizing RBD layering.  
+A much simpler example would be storage that can take around 5-10 minutes to snapshot a 200GB volume.  
+
+### Note: Data Volume cloning from snapshot source can work together with namespace transfer and size expansion  
+
+### Prerequisites
+  1) The source snapshot and target PVC share the same provisioner
+  2) The user creating the DataVolume has permission to create the `datavolumes/source` resource in the source namespace
+  3) Storage supports expansion (if the user attempts clone to larger target)
+
+### Flow Description
+- DataVolume is created with a Snapshot source
+- Check if restore from said snapshot is possible (as described in Prerequisites):
+- If possible:
+    * Create the restore PVC
+    * Set the claim reference of the PV to point to the new target PVC
```

Comment:
maybe mention that you change the claim ref of the restore PV from the restore PVC to the target PVC?

Reply:
I added a link to ns transfer doc